flake.nix

```
@@ -147,6 +147,22 @@
     ibc-go-v9-src.url = "github:cosmos/ibc-go/v9.0.0-beta.1";
     ibc-go-v9-src.flake = false;
 
+    ibc-go-v7-wasm-src = {
+      type = "github";
+      owner = "cosmos";
+      repo = "ibc-go";
+      ref = "08-wasm/release/v0.1.x+ibc-go-v7.3.x-wasmvm-v1.5.x";
+      flake = false;
+    };
+
+    ibc-go-v8-wasm-src = {
+      type = "github";
+      owner = "cosmos";
+      repo = "ibc-go";
+      ref = "modules/light-clients/08-wasm/v0.3.1%2Bibc-go-v7.4-wasmvm-v1.5";
```

Shouldn't this be ibc go v8.4 with wasm?

```suggestion
      ref = "modules/light-clients/08-wasm/v0.4.1%2Bibc-go-v8.4-wasmvm-v2.0";
```

---

Right, looks like I pasted the copied git tag to the wrong place. Should be fixed now.